pkg/config.go

```diff
@@ -19,7 +19,14 @@ type BaseConfig struct {
 
 type RDSConfig struct {
 	BaseConfig `yaml:"base,inline"`
-	Regions    []string `yaml:"regions"`
+	Regions    []string  `yaml:"regions"`
+	EOLInfos   []EOLInfo `yaml:"eol_info"`
```

Comment:
Since the EOLInfos stores in https://github.com/app-sre/aws-generated-data/blob/main/output/rds_eol.yaml. I may suggest just passing a URL here. So the exporter can read the file from github and get the latest info.
```suggestion
	EOLInfo string `yaml:"eol_info"`
```

Reply:
that content is synced via openshift-resources as config, the exporter shouldn't have outgoing requests to github

Reply:
The plan was to modify the aws-resource-exporter config map template within app-interface to pull the eol info from https://github.com/app-sre/aws-generated-data/blob/main/output/rds_eol.yaml so it will already be available to the exporter without any external calls.

Reply:
make sense. sorry for the noise.